ftw/colorbox/profiles/uninstall/Types/Folder.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object name="Folder"
+   meta_type="Factory-based Type Information with dynamic views"
```

Comment:
The folder of this file is `Types`, but it should be `types`.

Reply:
Goooood spot ! It wouldn't uninstall because of that!  - The default profile seemed to get away with the same error though.... (but I'll correct that too)